pkg/api/validation/validation.go

```diff
@@ -3524,7 +3524,7 @@ func ValidateResourceQuotaResourceName(value string, fldPath *field.Path) field.
 	if isLocalStorageResource(value) && !utilfeature.DefaultFeatureGate.Enabled(features.LocalStorageCapacityIsolation) {
 		return append(allErrs, field.Forbidden(fldPath, "ResourceEphemeralStorage field disabled by feature-gate for ResourceQuota"))
 	}
-	if len(strings.Split(value, "/")) == 1 {
+	if len(strings.Split(value, "/")) == 1 || strings.Contains(strings.ToLower(value), "alpha") {
```

Comment:
I'd recommend the following check instead.
`if len(strings.Split(value, "/")) == 1  || strings.Contains("kubernetes.io/")`

Reply:
Ok, updated it.